docs/source/model_design/vivarium_overview/vivarium_versus_other_model_types/index.rst

```diff
@@ -64,31 +74,114 @@ Over the duration of a given microsimulation run, simulant attributes are dynami
+
+In this section, we will elaborate on what differentiates microsimulation from other forms of IBM, such as agent-based simulation. 
+
+Advantages of microsimulation
++++++++++++++++++++++++++++++
+
+Correlation between risk exposures
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+At the individual level, risk exposures are frequently correlated. Examples include high body mass index and high fasting plasma glucose, tobacco smoking and alcohol use, and childhood height and weight.
+
+Usually, Vivarium assigns each risk exposure independently to simulants, such that each one follows the desired population-level univariate distribution, which frequently comes from GBD. In the common case of a dichotomous exposure, this means that each simulant (in the same age/sex/location group) has the same probability of exposure, which is equal to the prevalence in that population.
+
+Continuous values
+~~~~~~~~~~~~~~~~~
+
+e.g., allows the modeling of joint distribution between blood pressure and blood glucose levels -- this kind of thing is hard with categorical data characteristic of multiplication models
+
+Time-varying rates dependent on multiple factors
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+e.g., Multiple Myeloma model
+
+Incorporating uncertainty
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Our microsimulation framework is formulated to routinely propagate forward the uncertainty captured by these draws. Monte Carlo methods allow us to effectively incorporate different kinds of uncertainty into our models. 
+
+Monte Carlo microsimulation methods provide a useful approach to modeling stochastic uncertainty in a conceptually valid manner because chance – that is, reliance on randomly generated numbers – is used to govern movement between health states for individual simulants from a single computer-generated population, as opposed to comparing the role of chance across multiple populations with differing characteristics.
+
+In the context of healthcare estimates typically characterized by considerable variance across quantities of interest, the use of Monte Carlo methods to run simulations using iterative draws from probability distributions provides an effective means of incorporating parameter uncertainty. Simulation methods have the added advantage of reducing confounding variance between different scenarios by ensuring that the simulated population is identical across all scenarios analyzed. The only difference is the presence or absence of health interventions. As a further refinement of these methods, microsimulations represent each person in the population as an individual simulant with attributes variably assigned across simulants to reflect the population of interest. By incorporating heterogeneity at the level of the individual simulant, microsimulations are able to provide an additional degree of flexibility and detail (such as stratifying results according to demographic or epidemiological attributes) not available in population-level models. 
```

Comment:
> Simulation methods have the added advantage of reducing confounding variance between different scenarios by ensuring that the simulated population is identical across all scenarios analyzed. The only difference is the presence or absence of health interventions.

This part is about common random numbers, which as you've mentioned before aren't really an aspect of simulation methods in general but Vivarium in particular. I think this belongs in a different section.

Reply:
I think the last part (after the part quoted above) isn't about uncertainty and could be cut or moved to another section.